### PR TITLE
[Thor] Namespaced the vendored version of Thor and update to the v0.19.1...

### PR DESCRIPTION
**Pull request by segiddins**
... release

Closes https://trello.com/c/MJdUqrh3/59-namespace-vendored-thor.

### CODE REVIEW

File: lib/bundler/vendor/thor-0.19.1/lib/thor.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
-require 'set'
-require 'thor/base'
+require "set"
```
Comment (simi) [inline lib/bundler/vendor/thor-0.19.1/lib/thor.rb]: :question:

**Comment by simi** [comment]
Great work!

I'm just wondering if those cosmetic changes are really needed

**Comment by segiddins** [comment]
@simi I made no changes to source -- it's grabbing directly from the Thor release and just namespacing -- see the rake task I added

**Comment by simi** [comment]
Ahh, I missed that. Sorry.

**Comment by indirect** [comment]
Can you lowercase the directory name in vendor? It's traditional in ruby-land.

**Comment by segiddins** [comment]
> Can you lowercase the directory name in vendor? It's traditional in ruby-land.

done

**Comment by indirect** [comment]
Thanks!

**Pull request merged**